presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -871,6 +871,33 @@ public void testDistinctHaving()
                 "GROUP BY orderdate " +
                 "HAVING COUNT(DISTINCT clerk) > 1");
     }
+    @Test
+    public void testSelectFilter()
+            throws Exception
+    {
+        MaterializedResult actual = computeActual(
+                "SELECT SUM(X) FILTER(WHERE Y>4) FROM  " +
```

Comment:
SQL formatting:
* Space before `(` when following a keyword or table name (basically always, unless it's a function call)
* Lowercase function names, types and other identifiers
* Space around operators
* Space after comma in lists

Example:
```sql
SELECT sum(x) FILTER (WHERE y > 4)
FROM (VALUES (1, 3), (2, 4), (2, 4), (4, 5)) t (x, y)